tests/patch_attribute_testslide.py

```diff
@@ -110,9 +111,20 @@ def it_fails_if_attribute_is_a_class(self):
             def with_class_attributes(context):
                 context.merge_context("patching works")
 
-        @context.xexample
+        @context.example
         def it_fails_if_new_value_is_of_incompatible_type(self):
-            pass
+            with self.assertRaises(TypeCheckError):
+                self.patch_attribute(self.target, "typedattr", 123)
+
+        @context.example
+        def it_passes_if_new_value_is_of_incompatible_type_with_type_validation_false(
+            self,
+        ):
+            self.patch_attribute(self.target, "typedattr", 123, type_validation=False)
+
+        @context.example
+        def it_passes_if_new_value_is_of_matching_type(self,):
+            self.patch_attribute(self.target, "typedattr", "mocked")
```

Comment:
Suggestion: group all type validation tests under the same context:

```
@context.sub_context
def type_validation(context):
  @context.example
  ...
```

Reply:
done.